content/en/docs/score implementation/included/install-score-k8s-bash.md

```diff
@@ -0,0 +1,58 @@
+---
+title: "score-k8s install on macOS"
+description: "Installation for score-k8s on macOS."
+headless: true
+toc_hide: true
+---
+
+**1.** Download the latest compressed Tar file for your operating system from the [GitHub release page](https://github.com/score-spec/score-k8s/releases). By default, the tarball will be saved to your `~/Downloads` directory. If you choose to use a different location, you'll need to change that in the following steps.
```

Comment:
Not sure about this line. The tarball will be saved wherever you end up downloading it right?